packages/data-point-service/lib/cache-middleware.js

```diff
@@ -102,6 +102,20 @@ function shouldTriggerRevalidate (staleEntry, revalidationState) {
   )
 }
 
+function addRevalidationFlags (service, entryKey, revalidateTimeout) {
```

Comment:
these methods here are really just for readability purposes when being implemented, as others around them